rust_dev_preview/examples/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 # This should be kept in sync with https://github.com/awslabs/aws-sdk-rust#supported-rust-versions-msrv
 [toolchain]
-channel = "1.69.0"
```

Comment:
intentional?

Reply:
Yes, you or someone said we're tracking stable now. (Should be part of main, but it's fine here.)